dataland-frontend/tests/component/components/pages/CompanyCockpitPage.cy.ts

```diff
@@ -213,7 +220,7 @@ describe("Component test for the company cockpit", () => {
     });
   });
   it("Check for all expected elements from a logged-in data owner perspective with uploader-rights", () => {
-    mockRequestsOnMounted();
+    mockRequestsOnMounted(true);
     mountCompanyCockpitWithAuthentication(true, false, [KEYCLOAK_ROLE_UPLOADER], "mock-data-owner-id").then(() => {
```

Comment:
Könnt ihr nochmal die inputs prüfen. Manchmal ist es true, false oder gar nichts angegeben. Das macht es schwer den Test zu verstehen. Vll können auch pro Test Inputvariablen definiert werden die dann true oder false sind.

Reply:
Könnt ihr auch nochmal die Testbeschreibungen nachschärfen. Also zB der Test in Zeile 185 testet doch für einen nicht eingeloggten Benutzer den Fall, dass er eine Company ansieht die keinen DataOwner hat. Das sollte dann auch in der Beschreibung mit auftauchen.

Reply:
Der Testfall "nicht eingeloggt, aber die Firma ist geclaimed" also erwartet man das "Verified Icon" ist noch nicht abgedeckt oder habe ich etwas übersehen?

Reply:
habe den testfall hinzugefügt

Reply:
ok, danke